src/core/public/notifications/toasts/toasts_service.tsx

```diff
@@ -46,16 +48,18 @@ export class ToastsService {
     return this.api!;
   }
 
-  public start({ i18n, overlays, targetDomElement }: StartDeps) {
+  public start({ i18n, overlays, theme, targetDomElement }: StartDeps) {
     this.api!.start({ overlays, i18n });
     this.targetDomElement = targetDomElement;
 
     render(
       <i18n.Context>
-        <GlobalToastList
-          dismissToast={(toastId: string) => this.api!.remove(toastId)}
-          toasts$={this.api!.get$()}
-        />
+        <CoreThemeProvider theme$={theme.theme$}>
```

Comment:
we use the same pattern in all the integration points:
```jsx
<i18n.Context>
  <CoreThemeProvider theme$={theme.theme$}>
    ...
```
Maybe it makes sense to provide it as a single `CoreIntegrationPoint` component?